klock/src/commonMain/kotlin/com/soywiz/klock/internal/TimeZoneParser.kt

```diff
@@ -13,12 +13,13 @@ internal fun MicroStrReader.readTimeZoneOffset(tzNames: TimezoneNames = Timezone
     }
     if (reader.tryRead('Z')) return 0.minutes
     var sign = +1
-    if (!reader.tryRead("GMT") && !reader.tryRead("UTC")) return null
+    reader.tryRead("GMT")
+    reader.tryRead("UTC")
     if (reader.tryRead("+")) sign = +1
     if (reader.tryRead("-")) sign = -1
     val part = reader.readRemaining().replace(":", "")
-    val hours = part.substr(0, 2).padStart(2, '0').toIntOrNull() ?: 0
-    val minutes = part.substr(2, 2).padStart(2, '0').toIntOrNull() ?: 0
+    val hours = part.substr(0, 2).padStart(2, '0').toIntOrNull() ?: return null
+    val minutes = part.substr(2, 2).padStart(2, '0').toIntOrNull() ?: return null
```

Comment:
I know this was here before, but since you have changed it. Can you fix this?
I don't think the padStart before toIntOrNull makes sense at all. Probably the `padStart` was meant to be added to the `val part = ....padStart(...)`